### PR TITLE
Use the new syntax of imports and vararg splices

### PR DESCRIPTION
**Pull request by ShapelessCat**

### CODE REVIEW

File: src/main/scala/PatternMatching.scala [hunk 1]
```diff
@@ -38,7 +38,7 @@ object PatternMatching {
       def unapplySeq(name: String): Option[Seq[String]] = {
         val names = name.trim.split(" ")
         if (names.size < 2) None
-        else Some(names.last :: names.head :: names.drop(1).dropRight(1).toList)
+        else                Some(names.last :: names.head :: names.drop(1).dropRight(1).toList)
```
Comment (Philippus) [inline src/main/scala/PatternMatching.scala:41]: is this intentional?

Comment (ShapelessCat) [reply src/main/scala/PatternMatching.scala:41]: Yes. I try to align the results of each branch, here they are `None` and `Some`, to improve readability.

Comment (nicolasstucki) [reply src/main/scala/PatternMatching.scala:41]: That does not follow our coding style. We should keep it as it was.

Comment (ShapelessCat) [reply src/main/scala/PatternMatching.scala:41]: > That does not follow our coding style. We should keep it as it was.
OK. I can revert this change.

Comment (ShapelessCat) [reply src/main/scala/PatternMatching.scala:41]: > That does not follow our coding style. We should keep it as it was.

OK. I can revert this change.

**Review by nicolasstucki — APPROVED** [review]

**Pull request merged**